### ISSUE TITLE
Clarification in CircleLoss Documentation

### ISSUE THREAD

**Issue by ayhyap** [KevinMusgrave/pytorch-metric-learning#568]
After cross-referencing the paper, s_n and s_p are irrelevant to the loss function and should be removed from the documentation to avoid confusion (as it did to me for much longer than it should have).

EDIT: Also it appears the correct formula should be equation (6) in the paper, instead of equation (4).

**Comment by KevinMusgrave** [comment]
I think you're right. Thank you for pointing this out. I've updated the docs.
